include/modem/lte_lc.h

```diff
@@ -204,9 +207,158 @@ enum lte_lc_pdp_type {
 };
 
 enum lte_lc_pdn_auth_type {
-	LTE_LC_PDN_AUTH_TYPE_NONE = 0,
-	LTE_LC_PDN_AUTH_TYPE_PAP = 1,
-	LTE_LC_PDN_AUTH_TYPE_CHAP = 2,
+	LTE_LC_PDN_AUTH_TYPE_NONE	= 0,
+	LTE_LC_PDN_AUTH_TYPE_PAP	= 1,
+	LTE_LC_PDN_AUTH_TYPE_CHAP	= 2,
+};
+
+enum lte_lc_energy_estimate {
+	LTE_LC_ENERGY_CONSUMPTION_EXCESSIVE	= 5,
+	LTE_LC_ENERGY_CONSUMPTION_INCREASED	= 6,
+	LTE_LC_ENERGY_CONSUMPTION_NORMAL	= 7,
+	LTE_LC_ENERGY_CONSUMPTION_REDUCED	= 8,
+	LTE_LC_ENERGY_CONSUMPTION_EFFICIENT	= 9,
+};
+
+enum lte_lc_tau_triggered {
+	LTE_LC_CELL_IN_TAI_LIST		= 0,
+
+	/* Cell not in TAI list, TAU will be triggered. */
```

Comment:
The comments need to start with `/**` to be picked up by Doxygen. Applies to most new comments on struct members in the header file.